string ssid
int32 rssi
string security
